code/game/gamemodes/cult/hell_universe.dm

```diff
@@ -60,8 +60,9 @@ In short:
 
 	runedec += 9000	//basically removing the rune cap
 
-	for(var/turf/T in world)
-		T.update_lumcount(1, 255, 0, 0, 0)
+	spawn()
```

Comment:
Precaution, I made that loop in my last PR, and made a similar one this time for the sm cascade, but proc would sometimes have troubles because a lot of computing is done right at the start of the cascade, and updating the lumcount of every turf takes some time. Twice I saw the loop stop before doing all the turfs (resulting in half the station looking lit up (despite the lights being off) while the other half was properly lit). Having these updates happen in a different thread seemed to solve the issue.